sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/CodeGenInternal/EmailHookParameterPatch.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    internal partial class EmailHookParameterPatch
+    {
+        public IList<string> ToList { get; internal set; }
```

Comment:
Adding internal setter here and in `WebhookHookParameterPatch` to make code simpler.